controllers/machine_controller.go

```diff
@@ -92,6 +122,24 @@ func (r *MachineReconciler) SetupWithManager(mgr ctrl.Manager, options controlle
 	return nil
 }
 
+func (r *MachineReconciler) reconcileRequests(a handler.MapObject) []reconcile.Request {
```

Comment:
Maybe call this something like clusterToActiveMachines?

Reply:
+1